### PR TITLE
[relates #114] DB Connector e2e tests. part 1+part2

### PR DESCRIPTION
**Pull request by Stefan365**
this is part1 + part2 amended into 1 commit.
after PR #221 is merged, I will rebase this branch, so please ignore part 1.

### CODE REVIEW

File: ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/IntegrationFlowViewComponent.java [hunk 1]
```diff
@@ -19,6 +22,10 @@
 
 public class IntegrationFlowViewComponent extends SyndesisPageObject {
 
+	private static final class Link {
+		public static final By ADD_STEP = By.cssSelector("div.popover-content.popover-body > div > a");
```
Comment (lvydra) [inline ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/IntegrationFlowViewComponent.java]: I think you can get that link much more easier by By.linkText("Add Step").

File: ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/IntegrationFlowViewComponent.java [hunk 2]
```diff
@@ -28,6 +35,8 @@
 		public static final By ACTIVE_STEP = By.cssSelector("div[class='parent-step active']");
 		public static final By ACTIVE_STEP_ICON = By.cssSelector("p.icon.active");
 		public static final By DELETE = By.className("delete-icon");
+		public static final By STEP_INSERT = By.cssSelector("div.step.step-insert");
```
Comment (lvydra) [inline ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/IntegrationFlowViewComponent.java]: By.className("step-insert") should be enough too.

File: ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/ActionConfigureComponent.java [hunk 3]
```diff
@@ -8,13 +8,17 @@
...
 
-public class ActionConfigureComponent extends SyndesisPageObject {
+@Slf4j
+public abstract class ActionConfigureComponent extends SyndesisPageObject {
 
 	private static final class Element {
 		public static final By ROOT = By.cssSelector("syndesis-integrations-action-configure");
 	}
 
+	public abstract void fillInput(String value);
```
Comment (lvydra) [inline ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/ActionConfigureComponent.java]: I'm not sure if this is a good solution.

Comment (Stefan365) [reply ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/ActionConfigureComponent.java]: @lvydra this is a result of comments discussion of  .../src/test/java/io/syndesis/qe/pages/integrations/edit/ActionConfigureComponentInvokeSql.java in previous pull request #221 (the same topic). 
I think - if you have some specific idea how to do it - it's always best to give a proposal.

Comment (lvydra) [reply ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/ActionConfigureComponent.java]: @Stefan365 I know I read the discussion, but I don't think that you get Davids proposal right. He has been talking about solving problem with code duplicities on sql related ActionConfigureComponent. 
And there is nice spot for general ancestor.

Comment (Stefan365) [reply ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/ActionConfigureComponent.java]: @lvydra that was my above note about - if you have specific Idea, please come out with it and you prevent misunderstanding.

Comment (lvydra) [reply ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/ActionConfigureComponent.java]: @Stefan365 Well, I did it in last comment.

Comment (Stefan365) [reply ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/ActionConfigureComponent.java]: done

**Pull request merged**